README.md

```diff
@@ -47,6 +47,82 @@ Load the plugin:
 postgraphile --append-plugins @graphile/postgis
 ```
 
+#### Querying and mutating
+
+Using this table as example:
+
+```sql
+CREATE TABLE data (
+  id UUID PRIMARY KEY DEFAULT uuid_generate_v1mc(),
+  geom_point geometry(Point, 4326) default null
+);
+```
+
+In **queries** `geom_point` is represented as type `GeometryPoint`. Example:
+
+```graphql
+query {
+  allDatas {
+    nodes {
+      geomPoint {
+        geojson
+        srid
+        x
+        y
+      }
+    }
+  }
+}
+```
+
+In **mutations** `geom_point` is represented as type `GeoJSON`. Example:
+
+```graphql
+mutation ($id: UUID!, $geomPoint: GeoJSON!) {
+  updateDataById(
+    input: {
+      id: $id,
+      dataPatch: {
+        geomPoint: $geomPoint
+        }
```

Comment:
We have a bonus closing brace here, I think.
```suggestion
```

Reply:
^ Please commit this, I don't have permission to do so myself.

Reply:
Thanks, is corrected